shell/platform/embedder/embedder.h

```diff
@@ -217,6 +217,11 @@ typedef struct {
   // to respond to platform messages from the Dart application. The callback
   // will be invoked on the thread on which the |FlutterEngineRun| call is made.
   FlutterPlatformMessageCallback platform_message_callback;
+  // The path within |assets_path| at which the compiled Dart kernel file
+  // containing the |main| entry point can be found. The string can be
+  // collected after the call to |FlutterEngineRun| returns. The string must be
+  // NULL terminated.
+  const char* kernel_snapshot_path;
```

Comment:
Can you document the default please.

Reply:
Yikes! Good catch! Done.

Reply:
Also updated the (deprecated) comments on `main_path` and `packages_path` to refer to `kernel_snapshot_path` rather than `kernel_blob.bin` explicitly.